pajbot/bot.py

```diff
@@ -819,7 +822,8 @@ def on_clearchat(self, chatconn, event):
             else:
                 # permaban
                 # this sets timeout_end to None
-                user.timed_out = False
+                user.timed_out = True
```

Comment:
You can't set `timed_out` to true:

https://github.com/pajbot/pajbot/blob/37da06f4f18237486513e86eb2efcb80d5755bf4/pajbot/models/user.py#L173-L178

The only data we store for each user is a `timeout_end`, as you can imagine, you can't fit permabans into that sort of model, which is why we just clear the timeout_end field on permaban.
Reason is: There's just currently no use to knowing a user is banned (so why bother implementing it/storing it?). the only thing we use `timeout_end` is the paid timeout module (so you can stack timeouts).